src/components/organisms/Table/Sortable/SCell.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { FC, useContext } from 'react'
+import styled from 'styled-components'
+
+import { SortableTableSortDirection, SortableTableContext } from './SortableTable'
+import { Icon } from '../../../atoms/Icon/Icon'
+
+// T Y P E S
+export type SCellType = 'head' | 'body'
+export type SCellColor = 'primary' | 'light' | 'dark' | 'warning' | 'error' | 'success' | 'notice'
+
+export interface SCellProps {
+  type: SCellType
+  value?: string | boolean | number
+  color?: SCellColor
+  field?: string
+  sortable?: boolean
+}
+
+// C O M P O N E N T S
+
+export const SCell: FC<SCellProps> = (props) => {
+  const {
+    sortable = false,
+    field = '',
+    type,
+    value,
+    color,
+    children
+  } = props
+
+  const {
+    sortDirection,
+    sortedBy,
+    setSortDirection,
+    setSortedBy,
+    onSort
+  } = useContext(SortableTableContext)
+
+  const renderValue = (): string | boolean | number | JSX.Element | React.ReactNode => {
+    if (value) return value.toString()
+    if (field) return field.toString()
```

Comment:
Why is the `field` rendered a value?

If you remove it you can then replace all this function with `{value || children }` directly in the components `return`.

Reply:
`field` is given to specify a`key` in the data object. `field` doesn't matter for table data cells, but when i sort the table head cells, i will need unique `key` values.